src/components/auth/EnrollForm.tsx

```diff
@@ -0,0 +1,125 @@
+import { API_BASE_URL } from '@/config/config';
+import axios from 'axios';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+
+interface EnrollFormType {
+  email: string;
+  password: string;
+  checkPassword: string;
+  domain: string;
+}
+
+export const EnrollForm = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm<EnrollFormType>();
+  const navigate = useNavigate();
+  const handleEnrollSubmit = async (data: EnrollFormType) => {
+    try {
+      console.log(data, 'data');
+      const response = await axios.post(
+        `${API_BASE_URL}/dashboard/enrollClient`,
+        data
+      );
+      if (response.status === 200) {
+        navigate('/login');
+      }
+    } catch (err) {
+      console.error('가입요청 실패', err);
+      if (err instanceof Error) {
+        alert(err.message);
+      } else {
+        alert('회원가입 오류');
+      }
+    }
+  };
+
+  return (
+    <div className="flex-1">
+      <form onSubmit={handleSubmit(handleEnrollSubmit)} className="h-full">
+        <div className="flex flex-col justify-between h-full">
+          <h2 className="text-2xl font-bold text-center"> Enroll</h2>
+          <div className="flex items-center gap-2 w-full ">
+            <label className="w-[80px] text-sm">Email</label>
+            <input
+              {...register('email', {
+                required: { value: true, message: '이메일은 필수입니다 ' },
```

Comment:
```suggestion
                required: { value: true, message: '이메일은 필수입니다' },
```
이런 여백은 일부러 넣으시는 건가요?